src/featureMatching/__tests__/getOrientedFastKeypoints.test.ts

```diff
@@ -1,6 +1,6 @@
 import { getOrientedFastKeypoints } from '../getOrientedFastKeypoints';
 
-test('7x7 image, angle = 90°', () => {
+test.only('7x7 image, angle = 90°', () => {
```

Comment:
!

Reply:
Could we add an `eslint` rule or another check that would verify that there is no `test.only` in a file and either throw an error or a warning if there is? I find it easy to forget one while debugging.

Reply:
There is an eslint plugin for Jest: https://github.com/jest-community/eslint-plugin-jest#readme
We used to depend on it by default but it's not the case anymore. You can try to install it in this package and extend the recommended config

Reply:
The relevant rule is https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-disabled-tests.md

Reply:
> The relevant rule is https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-disabled-tests.md

It seems like that rule doesn't cover the case we're interested in:
![image](https://user-images.githubusercontent.com/17711389/211495882-49c34250-41a0-4dc2-83df-659196905744.png)

Reply:
you're right. I forgot the name of the rule you want, but it exists.

Reply:
Ok from what I see this would be the right rule: https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-focused-tests.md

Reply:
Could you help me out with the syntax in `.eslintrc.yml`?

I did something like this, but it doesn't work.

```yml
root: true
extends: [cheminfo-typescript, cheminfo-typescript/jsdoc]
plugins: [jest]
rules:
  '@typescript-eslint/restrict-template-expressions': off
  'jest/no-focused-tests': 'warn'
```

Reply:
I wasn't sure about the syntax for `.yml` files and eslint wouldn't run, but apparently I had just forgotten to install a package. It works now, sorry

Reply:
I had to install `yarn` to install the package, and it crated a `yarn.lock` file. Is this file useful?

Reply:
> I had to install yarn to install the package

Yarn is an alternative package manager to npm. There shouldn't be a need to install it.